README.md

```diff
@@ -0,0 +1,67 @@
+# bespin-job-watcher
+Allows users to receive job status notification updates for [bespin-api](https://github.com/Duke-GCB/bespin-api).
+Server subscribes to a rabbitmq exchange named 'job_status' and sends job status notifications to Websockets
+that have presented valid credentials for those jobs.
+
+## Requirements:
+- [nodejs 6.9.5+](https://nodejs.org/en/) - installed 
+- [Rabbitmq](http://www.rabbitmq.com/) - running instance
+- [bespin-api](https://github.com/Duke-GCB/bespin-api) - running instance
+
+## Setup:
+Create https/wss self signed certificates and install node modules.
```

Comment:
Without context of the discussion in the pull request, this is odd. It gives the impression that the certificate is needed to do the `npm install`.

Perhaps a sentence or two explaining the context. The certificates will be used by the web server and websocket server, Node/express make it easy to serve https, etc.